docs/faq.rst

```diff
@@ -330,6 +330,38 @@ needs to be ascertained and fixed.
 issues. We recommend to first run without ``--repair`` to assess the situation.
 If the found issues and proposed repairs seem right, re-run "check" with ``--repair`` enabled.
 
+How probable is it to get a hash collision problem?
+---------------------------------------------------
+
+If you noticed, there are some issues (:issue:`170` (**warning: hell**) and :issue:`4884`)
+about the probability of a chunk having the same hash as another chunk, making the file
+corrupted because it grabbed the wrong chunk. This is called the `Birthday Problem
+<https://en.wikipedia.org/wiki/Birthday_problem>`_.
+
+There is a lot of probability in here so, I can give you my interpretation of
+such math but it's honestly better that you read it yourself and grab your own
+resolution from that.
+
+Assuming that all your chunks have a size of :math:`2^{21}` bytes (approximately 2.1 MB)
+and we have a "perfect" hash algorithm, we can think that the probability of collision 
+would be of :math:`p^2/2^{n+1}` then, using SHA-256 (:math:`n=256`) and for example
+we have a 1,000 million chunks (:math:`p=10^9`) (a 1,000 million chunks would be about 2,100TB).
```

Comment:
we have another language / interpretation issue here:
x,yyy (en) == x.yyy (de) - thousands separator comma/point
x,yyy (de) == x.yyy (en) - decimal comma/point

so i'ld suggest better not using a thousands separator comma.

Reply:
im replacing the 1000000 to exp notation